code/modules/projectiles/gun.dm

```diff
@@ -40,7 +40,7 @@
 	var/weapon_weight = WEAPON_LIGHT
 	var/spread = 0						//Spread induced by the gun itself.
 	var/randomspread = 1				//Set to 0 for shotguns. This is used for weapons that don't fire all their bullets at once.
-
+	var/spawn_empty = TRUE
```

Comment:
I don't remember why i added the extra variable - other than I kinda was being sloppy - I can fix that though - although I see the potential for fixing a few syndicate weapons with your suggested changes of removing the second var (since I'm assuming they preset their pin to a special case) 

I guess nothing is ever as simple as it seem at first glance...also if you have spare time help me out by telling me about dupes - I try to avoid them but this github thingy is annoyingly new to me. Half the time they are the result of me uploading the root folder instead of it's contents when working with the upload files...I can /usually/ spot them because of the empty dirs but not always.